src/k8s/requests/v2v/constants.js

```diff
@@ -0,0 +1,9 @@
+export const CONVERSION_POD_TEMP_MOUNT_PATH = '/var/tmp';
+export const CONVERSION_POD_VDDK_MOUNT_PATH = '/opt/vmware-vix-disklib-distrib';
+
+export const CONVERSION_BASE_NAME = 'kubevirt-v2v-conversion';
+export const CONVERSION_GENERATE_NAME = `${CONVERSION_BASE_NAME}-`;
+
+export const CONVERSION_PROGRESS_ANNOTATION = 'v2vConversionProgress';
```

Comment:
We should not use `Annotations` for status info.

The `status` section of an object is better place.

Reply:
I just found out that it is not possible to write to status from mere pod. This needs to be done from a controller and currently we use only a pod.

Reply:
The "conversion pod" has access to the k8s api, it's just about setting privileges.
Annotations are definitely not good place to keep status info.
Whatever sets this information within Annotation now, can set it in the Status as well.

Reply:
I don't know the specifics, but there is nothing to do here until this is implemented by the conversion pod.

Reply:
After investigation using status is not possible. It is not possible to introduce new fields into the status structure -- we can only change existing fields. This is risky (data can be overwritten by other components) plus all info will be lost when the pod terminates.

Using labels is not good either as it has strict limitation on what the values can be and is not designed for storing general data.

Seems to me that using annotations is the only option.